src/App.tsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
-import Home from 'pages/Home';
 import GlobalStyles from 'styles/GlobalStyles';
+import MultiSelect from './components/atoms/MultiSelect';
 
 function App(): JSX.Element {
   return (
     <>
       <GlobalStyles />
-      <Home />
+      <MultiSelect isMulti />
```

Comment:
If the name of component is multiselect and multi prop is off, it could be confusing